test/helpers/mock_server.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'webrick'
+
+module MockServer
+  extend self
+
+  def start(hostname:, port:)
+    tid = Thread.new do
+      start_server(hostname, port)
+    end
+    poll_until_ready(hostname: hostname, port: port)
+    puts "Created test server #{tid}, port: #{port}"
```

Comment:
Was this intended to be left in?

Reply:
I think you had this file rendered pre https://github.com/Shopify/semian/pull/231/commits/d3c6d5e907ce1ee8b294b43cc7df4b6cb0fb7272

Reply:
Ah, well there's still a couple `puts` in the newer code as well.

Reply:
I changed this up a bit but yeah the `puts` for starting & killing the servers is intentional.